app/models.py

```diff
@@ -295,6 +295,7 @@ class User(Base, ModelMixin, UserMixin, PasswordOracle):
 
     FLAG_FREE_DISABLE_CREATE_ALIAS = 1 << 0
     FLAG_CREATED_FROM_PARTNER = 1 << 1
+    FLAG_FREE_OLD_ALIAS_LIMIT = 1 << 2
```

Comment:
If I understand correctly, after deployment of this PR, we're going to run a migration on `users` table to add this flag for all existing users right? In this case we can indeed merge this PR now and choose to run the query when the  change is public.

Reply:
:+1: